app/ide-desktop/lib/dashboard/src/components/AriaComponents/Text/useVisualTooltip.tsx

```diff
@@ -97,26 +116,33 @@ export function useVisualTooltip(props: VisualTooltipProps) {
   })
 
   return {
-    targetProps: aria.mergeProps<React.HTMLAttributes<HTMLElement>>()(hoverProps, { id }),
-    tooltip: isDisabled ? null : (
-      <Portal>
+    targetProps: aria.mergeProps<React.HTMLAttributes<HTMLElement>>()(targetHoverProps, { id }),
+    tooltip: state.isOpen ? (
```

Comment:
prefer inverting the condition, that way it's actually possible to know which condition the `null` corresponds to

Reply:
Not sure, on the one hand, it makes sense, on the other hand 
`!state.isOpen` is less readable and `!` is almost unnoticeable.

I think it's better to extract the tooltip element into a variable , wdyt?

Reply:
yeah, that makes sense too. idk about a variable though (it'd create unnecessary fibers), maybe an arrow function?

Reply:
Oh, I see what you mean. nice catch